src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -687,7 +687,7 @@ private static string GetMsBuildPathInPathVar()
             if (decimal.TryParse(userVersion, out parsedVersion))
             {
                 var adjustedVersion = (decimal)(((int)(parsedVersion * 10)) / 10F);
-                userVersionString = adjustedVersion.ToString("F1");
+                userVersionString = adjustedVersion.ToString("F1", CultureInfo.InvariantCulture);
```

Comment:
using decimal.tryparse seems wrong in the first place. is there a reason you can't use Version.TryParse ? https://msdn.microsoft.com/en-us/library/system.version.tryparse(v=vs.110).aspx

Reply:
actually we are using Version.Parse() later to parse userVersionString, I'm not sure the reason that we convert userVersion to userVersionString here, I'm guessing they are added for some scenario.